NewSpring/crete
# feat: Adds Test Groups to Users

## DESCRIPTION

![Screen Shot 2020-03-04 at 11 25 02 AM](https://user-images.githubusercontent.com/2659478/75900374-d1c11f80-5e0a-11ea-99e7-f407c25e7053.png)


### What does this PR do, or why is it needed?

Allows us to control app tests features per on a per user basis from Rock

### How do I test this PR?

Add yourself to the groups and make sure they show up with this query:

```
{
  currentUser {
    profile {
      featureGroups {
        name
      }
    }
  }
}
```

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
